.github/workflows/unit-test-widget.yml

```diff
@@ -0,0 +1,51 @@
+name: Generic widget unit test
+
+on:
+  workflow_call:
+    inputs:
+      test_command:
+        description: "Command to run widget unit tests. For example: yarn test"
+        required: true
+        type: string
+      skip_run:
+        description: "use to skip the workflow run"
+        type: boolean
+        default: false
+
+env:
+  WORK_DIRECTORY: "cumulocity-asset-properties-widget"
+
+jobs:
+  unit-tests:
+    runs-on: [self-hosted, ghiotapps]
+    if: ${{inputs.skip_run != true }}
+    defaults:
+      run:
+        working-directory: ${{ env.WORK_DIRECTORY }}
```

Comment:
no need to specify working-directory if it is same as the workspace (repo)

Reply:
removed it